openmetadata-ui/src/main/resources/ui/src/components/ServiceConfig/ConnectionConfigForm.tsx

```diff
@@ -74,7 +74,9 @@ const ConnectionConfigForm: FunctionComponent<Props> = ({
       data.hasOwnProperty('connection')
       ? ((data as DatabaseService | MessagingService | DashboardService)
           .connection.config as ConfigData)
-      : ({ pipelineUrl: (data as PipelineService).pipelineUrl } as ConfigData)
+      : ({
+          pipelineUrl: (data as PipelineService).connection.config?.hostPort,
```

Comment:
can you please add a fallback ? it will assign falsy value to a string

Reply:
This is expected to be undefined so api payload does not get empty value, also it will be changed in next patch, currently just pushing a patch to unblock other PRs from failing Maven CI.